.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ repos:
     rev: 5.12.0
     hooks:
       - id: isort
-        args: [--profile=black]
+        args: [--profile=black, --src=app]
```

Comment:
help isort find all our local modules to correctly sort imports

Reply:
Good find.